composer.json

```diff
@@ -26,11 +26,13 @@
     },
     "require-dev": {
         "friendsofphp/php-cs-fixer": "^2.15",
-        "phpunit/phpunit": "^8.3.5"
+        "phpunit/phpunit": "^8.3.5",
+        "ezsystems/ezplatform-code-style": "^1.0@dev"
```

Comment:
Keep the ordering alphabetical. Ideally we'd have `config.sort-packages: true` defined, but I'm not sure if `require` packages are already ordered or not.

Reply:
I used function, You suggested. Check it out now. :)